custom_components/integration_blueprint/entity.py

```diff
@@ -26,6 +26,6 @@ def __init__(self, coordinator: BlueprintDataUpdateCoordinator) -> None:
                 ),
             },
             name=coordinator.config_entry.runtime_data.integration.name,
-            model=coordinator.config_entry.runtime_data.integration.version,
+            model=str(coordinator.config_entry.runtime_data.integration.version),
```

Comment:
Can you adjust this PR to remove `name`, `model`, and `manufacturer` instead?
This is not a great set of defaults, and should not really be used by anyone.

Reply:
Sure 👍🏻

Reply:
Done in e3f6d92bf35f8c29b374225254f2feb15a32e37e